__tests__/reducers/index.test.js

```diff
@@ -80,8 +80,26 @@ describe('createReducer', () => {
   })
 })
 
+describe('setPublishError', () => {
+  it('adds error to editor state when no reason given', () => {
+    const newState = setPublishError(initialState.selectorReducer, {
+      type: 'PUBLISH_ERROR',
+    })
+
+    expect(newState.editor.serverError).toEqual('There was a problem saving the resource.')
+  })
+  it('adds error with reason to editor state', () => {
+    const newState = setPublishError(initialState.selectorReducer, {
+      type: 'PUBLISH_ERROR',
+      payload: { reason: 'publishing error msg' },
```

Comment:
I believe the payload here would be a string.

Reply:
Hrmm. I changed it and got this:

![image](https://user-images.githubusercontent.com/96775/62333116-0933ac00-b477-11e9-8ccb-27658c9173a1.png)

I tried to make it a string before (see comment train above).  What am I missing?